rosgraph_monitor/src/monitor.cpp

```diff
@@ -209,16 +226,23 @@ void RosGraphMonitor::track_node_updates(
       continue;
     }
 
-    auto [it, inserted] = nodes_.emplace(node_name, NodeTracking{});
+    NodeTracking tracking{};
```

Comment:
nonblocking: You could add a trivial constructor `NodeTracking(const std::string & name) : name(name) {}` so that you could keep this inline.

Reply:
great thanks